### ISSUE TITLE
`list-style`: duplicate report on `import { } from 'x'`

### ISSUE THREAD

**Issue by vladshcherbin** [eslint-stylistic/eslint-stylistic#1274]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

`import { } from 'x'` produces the same error twice:

```terminal
1:9  error  Should not have space(s) between '{' and '}'  @stylistic/list-style
1:9  error  Should not have space(s) between '{' and '}'  @stylistic/list-style
```

Cause:

`node.attributes` is an empty array here, so the `ImportDeclaration` visitor runs both the specifiers check and the `ImportAttributes` check. For zero-item `ImportAttributes`, `getLeftParen` picks the last `{` `}` token pair in
the statement (`pairs.at(-1)`), which in this code is the same specifier braces, so the pair gets checked twice.

Reproduction:

```ts
// eslint.config.mjs
import stylistic from '@stylistic/eslint-plugin'

export default [{
  plugins: { '@stylistic': stylistic },
  rules: { '@stylistic/list-style': ['error', { empty: 'never' }] }
}]

// repro.js
import { } from 'x'
```

Expected: one error.

Actual: two identical errors.

### Reproduction

Code is just one line, can make a repro if really needed

### System Info

```shell
@stylistic/eslint-plugin - 6.0.0-beta.6

oxlint - 1.79.0
oxlint-tsgolint - 7.0.2001
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests
